source/jwtd/jwt.d

```diff
@@ -3,18 +3,21 @@ module jwtd.jwt;
 import std.json;
 import std.stdio;
 import std.base64;
-
-import deimos.openssl.ssl;
-import deimos.openssl.pem;
-import deimos.openssl.rsa;
-import deimos.openssl.hmac;
-import deimos.openssl.err;
 import std.algorithm;
-
 import std.array : split;
 
+version (UseBotan) {
+	import jwtd.jwt_botan;
+}
+else {
+	import deimos.openssl.ssl;
+	import deimos.openssl.pem;
+	import deimos.openssl.rsa;
+	import deimos.openssl.hmac;
+	import deimos.openssl.err;
+}
 
-enum JWTAlgorithm {
+enum JWTAlgorithm : string {
```

Comment:
This gave me error about implicit conversion from string to int, so I added string as a base type